pkg/apis/kops/cluster.go

```diff
@@ -349,6 +352,18 @@ type ClusterSubnetSpec struct {
 	Type SubnetType `json:"type,omitempty"`
 }
 
+type EgressProxySpec struct {
+	HTTPProxy     HTTPProxySpec `json:"httpProxy,omitempty"`
+	ProxyExcludes string        `json:"excludes,omitempty"`
```

Comment:
I omit them by setting ProxyExcludes.  I was going to include documentation about this.